requirements.txt

```diff
@@ -5,3 +5,4 @@ Shapely
 smbus_cffi
 spidev
 aiortc
+llama-cpp-python
```

Comment:
It seems that upstream already publishes musl linux wheels, why does it need to be added here as well?

Reply:
Can you provide the link to the upstream musl wheels? PyPi still only provides a source tarball and no binaries at all.

Reply:
Hmm it seems that I looked wrongly. They are indeed not on Pypi. Maybe I've looked at the wrong package.

Have you contacted the author to ask if they are willing to publish Wheels?